zerver/webhooks/github/view.py

```diff
@@ -419,7 +420,7 @@ def get_status_body(helper: Helper) -> str:
     else:
         status = payload["state"].tame(check_string)
     return "[{}]({}) changed its status to {}.".format(
-        payload["sha"].tame(check_string)[:7],  # TODO
+        get_short_sha(payload["sha"].tame(check_string)),  # TODO
```

Comment:
We  got a TODO here back in #21581. @yuroitaki what was the context for this?

Reply:
I think that TODO is very likely to do this change and should be removed.

Reply:
Removed this TODO while merging; as a sidenote, the best practice is to always write TODO comments on their own line alongside a summary of what task needs to be done, precisely to avoid this sort of uncertainty :)